src/main/fc/settings.yaml

```diff
@@ -405,19 +405,19 @@ groups:
         max: INT16_MAX
       - name: maggain_x
         description: "Magnetometer calibration X gain. If 0, no calibration or calibration failed"
```

Comment:
Looks like the description should be changed. The value won't be 0 when no calibration has been done